File: December - 06/day 6.py
n=int(input("Position="))
list_A=list(map(int,input("Set of numbers: ").split()))
turn=input("Player going first: ")
c=0
while(list_A[0]!=0):
   list_A[0]=list_A[0]-1
   t=list_A[0]
   list_A[0]=list_A[n-1]
   list_A[n-1]=t
   c+=1 
    
if(turn=='Tanika' and c%2==1):
    print("Tanika wins the game!\nBob loses the game!")
elif(turn=='Tanika' and c%2==0):
    print("Bob wins the game!\nTanika loses the game!")
elif(turn=='Bob' and c%2==1):
    print("Bob wins the game!\nTanika loses the game!")
elif(turn=='Bob' and c%2==0):
    print("Tanika wins the game!\nBob loses the game!")
    

File: December - 06/day6.cpp
#include <bits/stdc++.h>
using namespace std;

int main() {
    cout<<"Enter size of numbers:";
    int size;cin>>size;
    cout<<"Enter pos:";
	int pos;cin>>pos;
    cout<<"Enter set of numbers:";
    vector<int>v(size);
    for(auto &x:v)cin>>x;
    
    cout<<"Player goinf first:";
    string s;
    cin>>s;
    int first_value = v[0];
    int pos_value = v[pos-1];
    
    int count;
    while(first_value>0){
        swap(--first_value,v[pos-1]);
        count++;
    }
    if(s=="Tanika"){
        if(count%2==0)
            cout<<"Tanika wins the game!\nBob loses the game!";
        else cout<<"Bob wins the game!\nTanika loses the game! ";
    }
    if(s=="Bob"){
        if(count%2==0)
            cout<<"Bob wins the game!\nTanika loses the game!";
        else cout<<"Tanika wins the game!\nBob loses the game! ";
    }

	return 0;
}
